CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ elseif (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 	add_definitions(-DRW_OSX)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
 	add_definitions(-DRW_FREEBSD)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+	add_definitions(-DRW_OPENBSD)
+	include_directories("/usr/X11R6/include")
+	set(OPENRW_PLATFORM_LIBS iconv)
```

Comment:
We depend on iconv on all platforms but will get rid of it soon (so this hack is fine).